configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,97 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
+  - dtc
+  - edk2
+  - fakechroot
+  - fakeroot
+  - fuse-sshfs
+  - gperftools
+  - gtk-vnc
+  - hexedit
+  - hivex
+  - icoutils
+  - ipxe
+  - kernel
+  - libdnet
+  - libguestfs
+  - libguestfs-winsupport
+  - libiscsi
+  - libmspack
+  - libnbd
```

Comment:
I would really like `libnbd` to appear in AppStream in RHEL 9.  (In RHEL 8 it is in the Virt module, but that was a mistake in hindsight).

Reply:
If it will still be maintained by you (or someone from sst_virtualization), then I would leave it here, as I don't think file differentiates between modular and non-modular content.

@asamalik - is there way for me to specify libnbd in the way Rich is asking ? Keep it in sst_virt, but not in the virt module.

Reply:
The BaseOS / Appstream split happens later. So as long as this is maintained by SST Virt, it stays here for now.
Same for `vhostmd` and `fuse-sshfs`.

`python-pyvmomi` is a dependency and will be pulled in by the content resolver tool. I will keep an eye for it after the initial resolver tool runs for Virt.